docs/learn-more/oauth.md

```diff
@@ -0,0 +1,87 @@
+# How to enable OAuth
+
+Following this guide, you will let the users to register and login using Google and Github OAUTH.
+
+- active `docker-compose.yml` (`~/.sourced/compose-files/__active__/docker-compose.yml`) should contain:
+    ```yml
+    version: '3.4'
+    services:
+    sourced-ui:
+      environment:
+        OAUTH_ENABLED_PROVIDERS: ${OAUTH_ENABLED_PROVIDERS}           # Comma separated list of available OAuth providers   (eg: github,google)
+        OAUTH_REGISTRATION_ROLE: ${OAUTH_REGISTRATION_ROLE}           # The role for the newly registered users using OAuth   'Admin'/'Alpha'/'Gamma'
+        OAUTH_GOOGLE_CONSUMER_KEY: ${OAUTH_GOOGLE_CONSUMER_KEY}       # OAuth provider consumer key (aka client_id)
+        OAUTH_GOOGLE_CONSUMER_SECRET: ${OAUTH_GOOGLE_CONSUMER_SECRET} # OAuth provider consumer secret (aka client_secret)
+        OAUTH_GITHUB_CONSUMER_KEY: ${OAUTH_GITHUB_CONSUMER_KEY}       # OAuth provider consumer key (aka client_id)
+        OAUTH_GITHUB_CONSUMER_SECRET: ${OAUTH_GITHUB_CONSUMER_SECRET} # OAuth provider consumer secret (aka client_secret)
+    ```
+
+In order to avoid some OAuth issues, make your instance of source{d} CE accessible from the Internet:
+- consider this example: `http://live.sourced.ce`
+    - `PROTOCOL` = `http`
+    - `HOST` = `live.sourced.ce`
+- you can use `ngrok` if running source{d} locally:
+    ```shell
+    $ ngrok http 8088
+    ```
```

Comment:
Have you verified this? I did not have any issues running on localhost with Google. Maybe this only happens with Github?